content/blog/2025-04-09-2024-recap.md

```diff
@@ -0,0 +1,100 @@
+    title: "Covering most of what the project was able to achieve in 2024"
+    image: /blog-images/2025/2024_recap.png
+---
+
+It's taken months for this post to get finalized, but it's because 2024 was another big year for us! We created a slew of great new things, overcame incredible challenges, and met and supported even more community members. I tried hard to keep this summary of 2024's work short, but it turns out you just can't even begin to highlight all of the impact of this vast and active community without it being long-winded.
+
+## First: thank you
+
+Before I get to the list of things the AlmaLinux community has accomplished this year, I want to share my gratitude.
+
+The support of our community ensures we continue to meet our goal of being the most stable, secure, and easy-to-engage-with enterprise Linux distribution. Whether it's funding support, helping spread the word, staffing booths at events, improving the translations on our website, helping each other on the forums or in chat, or just sporting an AlmaLinux pin, everything you do helps, and I can't thank you enough. 
+
+## New versions of AlmaLinux OS and more platforms to run it on!
+
+As our primary objective, continuing to provide stable, secure, Red Hat Enterprise Linux-compatible releases of AlmaLinux is our top priority. In 2024 we released [AlmaLinux 9.4](https://almalinux.org/blog/2024-05-06-announcing-94-stable/) and [AlmaLinux 9.5](https://almalinux.org/blog/2024-11-18-announcing-95-stable/), and the last minor version of 8 - [AlmaLinux 8.10](https://almalinux.org/blog/2024-05-28-announcing-810-stable/). We have easily continued to meet the expectations of AlmaLinux users by releasing our new version of AlmaLinux less than a week after RHEL.
+
+In case you were curious, there are two reasons we consistently release 6 days after RHEL. First: with our new build process, we have more work and testing to do before we are ready to release AlmaLinux to you. That work used to be complete in just over a day, but now it takes about 2 days. With new minor versions of RHEL typically being released on a Tuesday, we're typically ready to release late in the day on Thursday, but we care about our community too much to release on a Friday. :D
+
+We also announced our brand new [AlmaLinux OS Kitten](https://almalinux.org/blog/2025-01-07-almalinux-os-kitten-10-updates/), which is a development-focused version of AlmaLinux OS, and currently serves as the stable preview of AlmaLinux OS 10.
+
+{{< figure src="/blog-images/2025/kitten_2024_recap_blog.png" width="45%" alt="Screenshot of AlmaLinux OS Kitten Desktop" >}}
+
+While not many compared to the 1.4M+ of AlmaLinux OS 8 and 9, we already have a few thousand devices calling home running AlmaLinux OS Kitten 10! For a development branch, that's pretty cool. It also serves as the upstream for a couple projects, and is already helping projects based on AlmaLinux to get ready for AlmaLinux OS 10 - exactly what we were hoping for!
```

Comment:
```suggestion
While not many compared to the 1.4M+ of AlmaLinux OS 8 and 9, we already have a few thousand devices checking in with our public mirrorlist that are running AlmaLinux OS Kitten 10! For a development branch, that's pretty cool. It also serves as the upstream for a couple of projects, and is already helping projects based on AlmaLinux to get ready for AlmaLinux OS 10 - exactly what we were hoping for!
```
"calling home" sounds gross to me in this context, let's clarify what we mean by it and be more specific

also adds a missing preposition

Reply:
good catch, honestly. I should probably stop using that phrase.